torch/csrc/api/src/data/datasets/mnist.cpp

```diff
@@ -90,7 +90,8 @@ Tensor read_targets(const std::string& root, bool train) {
   expect_int32(targets, kTargetMagicNumber);
   expect_int32(targets, count);
 
-  auto tensor = torch::empty(count);
+  at::TensorOptions options;
+  auto tensor = torch::empty(count, options.dtype(at::kByte));
```

Comment:
Thanks a lot for the fix! Do you mind changing these two lines to just `auto tensor = torch::empty(count, torch::kByte);`? That's a more succinct way of writing this.

Reply:
Yes, indeed. I changed as you suggested. Thanks.